src/utils/bookmarkMigration.ts

```
@@ -0,0 +1,129 @@
+      logger.info('No existing bookmarks to migrate');
+      oldDb.close();
+      return;
+    }
+
+    // Get old bookmark data
+    const transaction = oldDb.transaction(['bookmarks', 'messages', 'chats'], 'readonly');
+    const bookmarkStore = transaction.objectStore('bookmarks');
+    const messageStore = transaction.objectStore('messages');
+    const chatStore = transaction.objectStore('chats');
+
+    const oldBookmarks = await new Promise<any[]>((resolve, reject) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)